.github/workflows/cd.yml

```diff
@@ -28,3 +33,4 @@ jobs:
           password: ${{ secrets.DOCKERHUB_DEVBOT_TOKEN }}
           dockerfile: Dockerfiles/Dockerfile
           buildargs: BUILD_DATE, VCS_REF
+          tags: "devel, ${{env.JINAD_VERSION}}"
```

Comment:
The openapi docs would depend on `JINAD_PREFIX` [(here)](https://github.com/jina-ai/jinad/blob/feat-add-pypi-setup/jinad/config.py#L13) which doesn't need to be semantically versioned, but should change with every major release.

Reply:
It will change only when the API is changed.